lib/LaTeXML/Package/graphics.sty.ltxml

```diff
@@ -232,19 +234,47 @@ DefConstructor('\graphicspath DirectoryList', sub {
 DefMacro('\includegraphics OptionalMatch:* [][] Semiverbatim',
   '\@includegraphics#1[#2][#3]{#4}');
 
+sub graphics_get_candidates {
+  my ($path, $kv) = @_;
+  $path = ToString($path); $path =~ s/^\s+//; $path =~ s/\s+$//;
+  $path =~ s/^("+)(.+)\g1$/$2/;                           # unwrap if in quotes
```

Comment:
One of these days we should have a `LaTeXML::Util::Pathname` helper for these path cleaning tricks.